graphistry/embed_utils.py

```diff
@@ -319,7 +316,7 @@ def embed(
         if inplace:
             res = self
         else:
-            res = self.bind()
+            res = self.bind() # type: ignore
```

Comment:
more of these ignores...

Reply:
because bind returns a Plottable and we want it to see it's own `_private_mixin_funcs`

Reply:
so do the cast?